boards/arm/twr_ke18f/doc/index.rst

```diff
@@ -0,0 +1,211 @@
+.. _twr_ke18f:
+
+NXP TWR-KE18F
+#############
+
+Overview
+********
+
+The TWR-KE18F is a development board for platform for the NXP Kinetis
```

Comment:
How about:
```
The TWR-KE18F is a development board for NXP Kinetis
KE1xF 32-bit MCU-based platforms.  The onboard OpenSDAv2 serial and
debug adapter, running an open source bootloader, offers options for
serial communication, flash programming, and run-control debugging.
```

Reply:
@dbkinder I appreciate your review. I have made the suggested changes.